buildSrc/src/main/java/se/nbis/lega/deployment/DeployStackTask.java

```diff
@@ -1,18 +1,26 @@
 package se.nbis.lega.deployment;
 
-import org.gradle.api.tasks.TaskAction;
-
 import java.io.IOException;
+import java.util.Formatter;
 import java.util.Map;
+import java.util.Set;
+import org.gradle.api.tasks.TaskAction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class DeployStackTask extends LocalEGATask {
+    private static final Logger logger = LoggerFactory.getLogger(DeployStackTask.class);
 
     private String composeFile;
     private Map<String, String> environment;
 
     @TaskAction
     public void run() throws IOException {
         environment.putAll(getMachineEnvironment(machineName));
+        Set<String> keySet = environment.keySet();
```

Comment:
Not sure if we want to print all the secrets to the outer world (Jenkins builds are public). If there's no special use for it, we should probably hide it.